notebooks/triton/deploy-bidaf-aks.ipynb

```diff
@@ -0,0 +1,259 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "from azureml.core.model import Model\n",
+    "\n",
+    "model_path = prefix.joinpath(\"models\", \"triton\")\n",
+    "\n",
+    "model = Model.register(\n",
+    "    model_path=model_path,\n",
+    "    model_name=\"bidaf-9-example\",\n",
+    "    tags={\"area\": \"Natural language processing\", \"type\": \"Question-answering\"},\n",
+    "    description=\"Question answering from ONNX model zoo\",\n",
+    "    workspace=ws,\n",
+    "    model_framework=Model.Framework.MULTI,\n",
+    ")\n",
+    "\n",
+    "print(model.name, model.description, model.version)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Deploy webservice\n",
+    "\n",
+    "In this case we deploy to a pre-created [AksCompute](https://docs.microsoft.com/python/api/azureml-core/azureml.core.compute.aks.akscompute?view=azure-ml-py#provisioning-configuration-agent-count-none--vm-size-none--ssl-cname-none--ssl-cert-pem-file-none--ssl-key-pem-file-none--location-none--vnet-resourcegroup-name-none--vnet-name-none--subnet-name-none--service-cidr-none--dns-service-ip-none--docker-bridge-cidr-none--cluster-purpose-none--load-balancer-type-none-) called \"aks-gpu-deploy,\" but for other options, see [our documentation](https://docs.microsoft.com/azure/machine-learning/how-to-deploy-and-where?tabs=azcli)\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": [
+     "outputPrepend"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "from azureml.core.webservice import AksWebservice\n",
+    "from azureml.core import Environment\n",
+    "from azureml.core.model import InferenceConfig\n",
+    "\n",
+    "service_name = \"triton-bidaf-9\"\n",
+    "\n",
+    "config = AksWebservice.deploy_configuration(\n",
+    "    compute_target_name=\"aks-gpu-deploy\",\n",
+    "    gpu_cores=1,\n",
+    "    cpu_cores=1,\n",
+    "    memory_gb=4,\n",
+    "    auth_enabled=False,\n",
+    ")\n",
+    "\n",
+    "service = Model.deploy(\n",
+    "    workspace=ws,\n",
+    "    name=service_name,\n",
+    "    models=[model],\n",
+    "    deployment_config=config,\n",
+    "    overwrite=True,\n",
+    ")\n",
+    "\n",
+    "service.wait_for_deployment(show_output=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Test the webservice"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import json\n",
+    "import sys\n",
+    "\n",
+    "# Using a modified version of tritonhttpclient for Preview, PR is out for review\n",
+    "# https://github.com/triton-inference-server/server/pull/2047\n",
+    "sys.path.insert(1, prefix.joinpath(\"code\", \"deployment\", \"triton\").__str__())\n",
```

Comment:
is this sys.path magic needed? what's it doing?

Reply:
Adding `tritonhttpclient` to the path so it can be imported as a module